ONSdigital/eq-questionnaire-runner
# [BUG]Make survey title generic

### What is the context of this PR?
The survey title is currently set to ONS Business Survey even when no cookie exists. This PR changes it so the generic title, which is "ONS Surveys" which no cookie exist
Added a function called `get_survey_titie `to `template_helpers.py` 

### How to review 
- Locally run runner
- Start any survey, and remove the session using developer tools
- Refresh the page and it should return an error page with the title "ONS Surveys"
- Check updated unit-tests due to these changes. Especially `test_correct_theme_in_context` because the only title it will return is `ONS Surveys`

### Checklist
